rotkehlchen/db/filtering.py

```diff
@@ -842,6 +843,20 @@ def prepare(self) -> tuple[list[str], list[Any]]:
 T_HistoryFilterQuery = TypeVar('T_HistoryFilterQuery', bound='HistoryBaseEntryFilterQuery')
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class HistoryEventCustomizedOnlyJoinsFilter(DBFilter):
+    """This join finds customized history events (exclusively)."""
+    def prepare(self) -> tuple[list[str], list[Any]]:
+        query = (
+            'INNER JOIN history_events_mappings '
+            'ON history_events_mappings.parent_identifier = history_events.identifier '
+            'WHERE history_events_mappings.name = ? AND history_events_mappings.value = ?'
+        )
+        bindings: list[Union[str, int]] = []
+        bindings.extend([HISTORY_MAPPING_KEY_STATE, HISTORY_MAPPING_STATE_CUSTOMIZED])
```

Comment:
You are always extending it so you can just define it directly
```suggestion
        bindings: list[Union[str, int]] = [HISTORY_MAPPING_KEY_STATE, HISTORY_MAPPING_STATE_CUSTOMIZED]
```

Reply:
Updated as suggested.